Dockerfile

```diff
@@ -5,42 +5,66 @@ FROM timescale/timescaledb:${TIMESCALEDB_VERSION}-${PG_VERSION_TAG} as builder
 MAINTAINER Timescale https://www.timescale.com
 ARG PG_VERSION_TAG
 
-RUN set -ex \
-    && apk add --no-cache --virtual .build-deps \
-                coreutils \
-                dpkg-dev dpkg \
-                gcc \
-                libc-dev \
-                make \
-                util-linux-dev \
-                clang \
-		llvm \
-                git \
-                llvm-dev clang-libs \
-		bison \
-		dpkg-dev dpkg \
-		flex \
-		gcc \
-		libc-dev \
-		libedit-dev \
-		libxml2-dev \
-		libxslt-dev \
-		linux-headers \
-	 	clang g++ \
-		make \
-		openssl-dev \
-		perl-utils \
-		perl-ipc-run \
-		util-linux-dev \
-		zlib-dev \
-		icu-dev
+RUN set -ex; \
+    apk add --no-cache --virtual .build-deps \
+        coreutils \
+        dpkg-dev dpkg \
+        gcc \
+        libc-dev \
+        make \
+        util-linux-dev \
+        clang \
+        llvm \
+        git \
+        llvm-dev clang-libs \
+        bison \
+        dpkg-dev dpkg \
+        flex \
+        gcc \
+        libc-dev \
+        libedit-dev \
+        libxml2-dev \
+        libxslt-dev \
+        linux-headers \
+        clang g++ \
+        make \
+        openssl-dev \
+        perl-utils \
+        perl-ipc-run \
+        util-linux-dev \
+        zlib-dev \
+        icu-dev
 
-ENV RUSTUP_HOME=/usr/local/rustup \
-    CARGO_HOME=/usr/local/cargo \
-    PATH=/usr/local/cargo/bin:$PATH \
-    RUST_VERSION=1.53.0 \
-    # prevents some issues with openssl
-    RUSTFLAGS="-C target-feature=-crt-static"
+RUN set -ex; \
```

Comment:
I honestly can't answer this. I suspect that @cevian would prefer that we don't remove it.

Reply:
@cevian Do you know what purpose pgextwlist serves in this Dockerfile?

Reply:
After removing it, I'm not seeing any differences as a result, so I'm pretty convinced it isn't serving any purpose currently.

Reply:
Promscale's `sql_permissions.md` mentions the following:

# Using a non-superuser owner with pgextwlist

The owner user of Promscale need to be able to install
and upgrade the promscale and timescaledb PostgreSQL extensions.

Normally extension installation and upgrade requires superuser permissions. To avoid
using superuser, you could use the [pgextwlist](https://github.com/dimitri/pgextwlist) extension with the following PostgreSQL
config:
```
local_preload_libraries=pgextwlist
extwlist.extensions=promscale,timescaledb
```

Reply:
I know that we were planning on changing this by marking the extension as trusted.

Reply:
We're installing it in the build stage though, not the final image that users will use. If we do want to support this though, I'd suggest we add it to the base timescaledb image, since that feels more appropriate rather than having it only in the image that has the Promscale extension.

Reply:
doh. I didn't even realise that. I may have broken that in my refactoring.

Reply:
It looks like pgextwlist _would_ be in the final image in the original Dockerfile, because we were copying all of `/usr/local/lib/postgresql` and `/usr/local/share/postgresql` into the final image, and I'm assuming pgextwlist was being installed in those. With the changes made to copy from the directories generated by `pgx` though, which don't contain that extension AFAICT, they are no longer being copied into the final image.

Reply:
Yup, I just verified that I broke that. pgextwlist.so was present on the image before this refactoring.

Reply:
Since we're planning to mark the extension as trusted anyway, it seems we probably don't need it, at least based on the docs. @cevian do you know if there is any other reason why we would still require pgextwlist in that scenario?

Reply:
pgextwlist should be necessary for some tests from the connector and is necessary for the permission instructions cited above. Since we aren't marking the extension trusted yet, I don't think we can remove it. Although after it's trusted then we can

Reply:
Makes sense, I was under the impression we aren't planning another release of the extension until our other changes are in place though, including marking the extension trusted, so we should be in the clear to remove it. The previous docker images containing pgextwlist and the latest release of the extension are still available (and presumably used by promscale for its test suite), so these changes only affect current development and future releases.